package/Dockerfile-windows.agent

```diff
@@ -25,4 +25,8 @@ RUN $URL = 'https://{0}/fleet/{1}/fleetagent-windows-amd64.exe' -f $env:RELEASES
     Write-Host ('Downloading dapper from {0} ...' -f $URL); \
     [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
     Invoke-WebRequest -UseBasicParsing -OutFile c:/fleet/fleetagent-windows.exe -Uri $URL;
+FROM mcr.microsoft.com/windows/nanoserver:${SERVERCORE_VERSION}
+USER ContainerAdministrator
+COPY --from=builder /fleet /fleet
+COPY --from=builder /git /git
 CMD ["C:\fleet\fleetagent-windows.exe"]
```

Comment:
I believe we have to copy the `agent` as well. Three copies?

Reply:
its not there

Reply:
Miscommunication: I though `/fleet` was a binary copy for some reason. It's a directory copy and contains the `agent`. Resolving this.